README.md

```diff
@@ -619,6 +619,7 @@ Table of Contents
   * [checklyhq.com](https://checklyhq.com) - Open source E2E / Synthetic monitoring and deep API monitoring for developers. Free plan with 5 users and 50k+ check runs.
   * [circonus.com](https://www.circonus.com/) — Free for 20 metrics
   * [cloudsploit.com](https://cloudsploit.com) — AWS security and configuration monitoring. Free: unlimited on-demand scans, unlimited users, unlimited stored accounts. Subscription: automated scanning, API access, etc.
+  * [cronrobot.io](https://www.cronrobot.io) — Open source monitoring and automation platform for your websites. Free tier for personal use. Can be used with our cloud solution or self-hosted.
```

Comment:
unfortunately at this point there is either "coming soon" free or "free tier while in beta" neither qualifies for the list, once you are sure you'll offer a long term free tier and has its detailed published we can revisit this one